src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs

```diff
@@ -34,9 +34,7 @@ public Http3LoopbackServer(QuicImplementationProvider quicImplementationProvider
                     EnabledSslProtocols = options.SslProtocols,
                     ApplicationProtocols = new List<SslApplicationProtocol>
                     {
-                        new SslApplicationProtocol("h3-31"),
-                        new SslApplicationProtocol("h3-30"),
-                        new SslApplicationProtocol("h3-29")
+                        new SslApplicationProtocol("h3")
```

Comment:
Should we add Http3 to

https://github.com/dotnet/runtime/blob/361184ff71b53ed8565bf8a7ecf3d4809c5ea689/src/libraries/System.Net.Security/ref/System.Net.Security.cs#L121-L122

(fine IMHO if we do it as follow-up)

Reply:
I was reluctant to bring public API change as we are now past Feature-Complete date.

Reply:
We should just make sure we have a 7.0 issue filed on this. I don't think we care about it for 6.0.

Reply:
It's here https://github.com/dotnet/runtime/issues/1293